ios/brave-ios/Sources/Brave/Frontend/UserContent/UserScripts/Sandboxed/MainFrame/AtDocumentStart/ReaderModeScript.js

```diff
@@ -35,7 +35,7 @@ function checkReadability() {
     }
     
     if(!isProbablyReaderable(document)) {
-      webkit.messageHandlers.readerModeMessageHandler.postMessage({Type: "ReaderModeStateChange", Value: "Unavailable"});
+      webkit.messageHandlers.readerModeMessageHandler.postMessage({"securityToken": SECURITY_TOKEN, "data": {Type: "ReaderModeStateChange", Value: "Unavailable"}});
```

Comment:
Actually, can't a web page re-define `webkit.messageHandlers.readerModeMessageHandler.postMessage` and leak the token?

Reply:
indeed this was the case with https://hackerone.com/reports/1668723, can you check it hasn't regressed?

Reply:
Ah, this is a sandboxed script, it is not injected into the page content world, so we don't even really need a security token here, so it's fine